lib/groovehq/resource_collection.rb

```diff
@@ -25,15 +27,25 @@ def initialize(client, data)
           prev: {
             href: meta_data["pagination"]["prev_page"]
           }
-        }.with_indifferent_access
+        }
       end
 
       @data = OpenStruct.new(meta: meta_data, collection: collection)
       @rels = parse_links(links)
+      @options = options.with_indifferent_access
     end
 
-    def each(&block)
+    def each
+      return enum_for(:each) unless block_given?
+
       collection.each { |item| yield item }
+
+      rel = @rels[:next] or return self
+      coll = rel.get(@options.except(:page))
```

Comment:
Short for `collection` since it represents the next instance of a `ResourceCollection` in the paginated set.  Do you prefer `collection` or perhaps `other`?